config.yml

```diff
@@ -78,4 +80,4 @@ rest_runner:
     translation_enabled: True
     translation_model: facebook/seamless-m4t-v2-large # facebook/seamless-m4t-v2-large
     translation_device: cuda # cpu | cuda
-    translation_method: segmented # segmented | full
+    translation_default_method: "segmented" # "segmented" | "full"
```

Comment:
I am confused. In the local config this is only present at the top. Here it is present twice :thinking:

Reply:
mb, ty